File: Assignment 26/program4.java
////////////////////////////////////////////////////////////////////////////////
//
//  Required Package
//
////////////////////////////////////////////////////////////////////////////////

import java.util.*;

////////////////////////////////////////////////////////////////////////////////
//
//  Class Name    : class pattern
//  Description   : It is used to Display pattern 
//  Input         : Intiger  
//  OutPut        : Pattern  
//  Author        : Pratibha Sagar Pawar
//  Date          : 21 Nov 2025
//      
////////////////////////////////////////////////////////////////////////////////
class pattern
{
    public void Display(int iNo)
    {
        int iCnt = 0;
        int iCount = 0;
       
         iCount = 1;
        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
           if(iCnt % 2 ==0)
           {
            System.out.print(iCount+"\t*\t");
            iCount++;
           }
           else
           {
            System.out.print("#\t");
            
           }
        }
        System.out.println();    

    }
}
///////////////////////////////////////////////////////////////////////////////
//
//  Entery point method for the application 
// 
////////////////////////////////////////////////////////////////////////////////
class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int iValue = 0;

        System.out.println("Enter the frequency : ");
        iValue = sobj.nextInt();

        pattern pobj = new pattern();
        pobj.Display(iValue);
    }
}
//  End of main method
////////////////////////////////////////////////////////////////////////////////
//
//  Input  : 4
//  Output : #  1   *   #   2   *    #   3   *   #   4   *
//           1  2   3   4   5   6    7   8   9   10  11  12   
//
////////////////////////////////////////////////////////////////////////////////

File: Assignment 34/program4.c
////////////////////////////////////////////////////////////////////////////////
//
//  Required Heder Files
//
////////////////////////////////////////////////////////////////////////////////
#include<stdio.h>
#define TRUE 1
#define FALSE 0

////////////////////////////////////////////////////////////////////////////////
//
//  Function Name : ChkSpecial
//  Description   : It is used to check special symbol 
//  Input         : character  
//  OutPut        : character 
//  Author        : Pratibha Sagar Pawar
//  Date          : 29 Nov 2025
//      
////////////////////////////////////////////////////////////////////////////////

typedef int BOOL;

BOOL ChkSpecial(char ch)
{
    if(ch >= 'A' && ch<= 'Z'|| ch >= 'a' && ch<= 'z' || ch >= '0' && ch<= '9')    
    {
        return FALSE;
    }
    else
    {
        return TRUE;
    }

}

////////////////////////////////////////////////////////////////////////////////
//
//  Entery point function for the application 
// 
////////////////////////////////////////////////////////////////////////////////

int main()
{
    char cValue = '\0';
    BOOL bRet = FALSE;

    printf("Enter the character : ");
    scanf("%c",&cValue);

    bRet = ChkSpecial(cValue);

    if(bRet == TRUE)
    {
        printf("It is special Character");
    }
    else
    {
       printf("it is not a special character");

    }
    return 0;
}
//  End of main Function
////////////////////////////////////////////////////////////////////////////////
//
//  Input : A    Output : it is not a special  character
//  Inpur : #    Output : it is a special character  
//
////////////////////////////////////////////////////////////////////////////////
